controllers/component_build_controller_pac.go

```diff
@@ -148,6 +156,162 @@ func (r *ComponentBuildReconciler) ProvisionPaCForComponent(ctx context.Context,
 	return mrUrl, nil
 }
 
+func getHttpClient() *http.Client {
```

Comment:
nitpick: `createHttpClient` or `newHttpClient`

Reply:
current name seems more appropriate, since it really returns client